test/vm.c

```diff
@@ -50,7 +50,7 @@ void test_vm_alloc(void)
 			break;
 		}
 
-		_page_free(p);
+		vm_pageFree(p);
```

Comment:
This technically changes code logic, as we use synchronized version of `pageFree` now. But it is just a test, so I'm not sure whether we should mark this change as bugfix/new feature or leave as chore.

Reply:
phoenix-rtos/phoenix-rtos-project/issues/1080 is fixed there, so maybe mark the change as bugfix.